backend/slack_commands/clients/multiagent.py

```
@@ -0,0 +1,173 @@
+            for bp in raw
+        ]
+
+    def session_exists(self, session_id: str, user_name: str) -> bool:
+        try:
+            resp = requests.get(
+                f"{self._base_url}/api/sessions/session.status.get",
+                params={"sessionId": session_id},
+                headers=self._auth_headers(user_name),
+                timeout=5,
+            )
+            return resp.status_code == 200
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)